tests/braynsTestData.cpp

```diff
@@ -222,6 +222,31 @@ BOOST_AUTO_TEST_CASE(render_circuit_with_particle_renderer)
                                  brayns.getEngine().getFrameBuffer()));
 }
 
+BOOST_AUTO_TEST_CASE(render_circuit_with_proximity_renderer)
```

Comment:
This test doesn't use a circuit here. We should move all the tests that do not need BBPTestDaa to a separate file.

Reply:
I agree, but will do a separate PR for that.